src/components/NcCounterBubble/NcCounterBubble.vue

```diff
@@ -212,7 +212,7 @@ const originalCountAsTitleIfNeeded = computed(() => {
 	text-align: center;
 	line-height: var(--counter-bubble-height); // Expand line-height to full height to center text vertically
 	padding: 0 calc(1.5 * var(--default-grid-baseline));
-	border-radius: var(--border-radius-pill);
+	border-radius: calc(var(--counter-bubble-height) / 2);
```

Comment:
```suggestion
	border-radius: var(--border-radius-pill);
```

Reply:
Pill is deprecated

Reply:
What is the replacement for rounded?

Reply:
Just use half of the height?

Reply:
Why not use universal value without calculations?

Reply:
1lh also works here as a simple value, but I didn't get why remove universal pill variable

Reply:
And half height is the same as just height here

Reply:
Sure we could also just use
```suggestion
	border-radius: 0.5lh;
```